README.md

```diff
@@ -26,16 +29,26 @@ To obtain coverage results via the console you need to dump the console output i
 
 The console output then needs to be processed into an lcov report. `lib-embedded-gcov` provides the `gcov_process.sh` script for that, which is essentially a wrapper around the tools provided by `embedded-gcov`.
 
-Before executing the script make sure you have the required dependencies installed, that is:
-- dox2unix
-- lcov
-
 With dependencies installed, execute the script as:
 ```
 lib-embedded-gcov/scripts/gcov_process.sh <build_directory> <console_log>
 ```
 
-The script will generate an lcov report and provide a link as shown below.
+### Binary File Output
+To obtain coverage results via a binary file, you will need to provide a file system and also implementations for the following functions: `open()`, `write()`, `close()`. The easiest way to do this is to include musl as a library and use 9pfs as the filesystem. Possible QEMU configuration:
```

Comment:
Until the vfs library provides its own documentation, perhaps it would be worth to provide brief instructions on how to configure Unikraft with 9pfs, ie Kconfig and `mkdir rootfs`, to speed up the process for those who are not familiar with 9pfs.

Reply:
Modified README.md. Take a look when you can and maybe we can further improve it.

Reply:
I created a reworked version of the README, in an attempt  to make it a bit more terse and use a unified language style: https://github.com/michpappas/lib-embedded-gcov-1/tree/update-readme